task-requests/style.css

```diff
@@ -85,11 +85,79 @@ body {
 .container__body {
   margin-top: 2.5rem;
 }
-.sort-filters {
+
+.filters__header {
+  display: none;
+  gap: 1rem;
+  align-items: center;
+  margin-bottom: 8px;
+}
+
+.filters__header > h2 {
+  margin: 0;
+  font-size: 1.25rem;
+}
+
+.filters__header > button {
+  background: none;
+  cursor: pointer;
+  padding: 3px 8px;
+  border: 1px solid var(--light-gray-color);
+  border-radius: 7px;
+}
+
+.filters__header > button:hover {
+  background-color: var(--red-color);
+  color: var(--white);
+}
+
+.sort-filters__badges {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  flex-wrap: wrap;
+}
+
+.badge {
   display: flex;
-  justify-content: end;
   align-items: center;
+  justify-content: center;
+  gap: 8px;
+  padding: 10px 1rem;
+  background-color: var(--light-gray-color);
+  border-radius: 7px;
+  cursor: pointer;
+  border: 1px solid var(--light-gray-color);
+}
```

Comment:
styling for badges could be enhanced by adding transitions or animations for smoother visual effects 
` transition: background-color 0.3s ease, color 0.3s ease;`

Reply:
thanks for the suggestion, added the transition.